src/Storages/System/StorageSystemTableViews.cpp

```diff
@@ -0,0 +1,47 @@
+#include <Storages/System/StorageSystemTableViews.h>
+#include <DataTypes/DataTypeString.h>
+#include <Access/ContextAccess.h>
+#include <Interpreters/Context.h>
+
+namespace DB
+{
+
+class Context;
+
+NamesAndTypesList StorageSystemTableViews::getNamesAndTypes()
+{
+    return {
+        {"database", std::make_shared<DataTypeString>()},
```

Comment:
I am not sure if naming is good enough.
From my perspective better name for table is system.views.
For columns table system.views should have same naming as system.tables.
`database` - database of view
`name` - name of view
`table` - view table
`table_database` - view table database.